app/modules/encounters/models.py

```diff
@@ -5,7 +5,7 @@
 """
 
 from app.extensions import db, FeatherModel
-
+from app.modules.sightings.models import Sighting
```

Comment:
Importing models from other models at a module scope caused circular dependencies. 
JP changed them all to be function level imports on the rest of the code.

Reply:
Cool, I can see that pattern now. Resolved in this case and will remember for later.

Reply:
Yes, the design decision is we want config.py to control and define the import order of modules.  This defines how database tables are created and the SQLAlchemy ORM structures are registered.  If we move the imports out of module level and into function and class level, we can preserve the correct order.